assets/apps/dashboard/src/Components/ModuleCard.js

```diff
@@ -38,7 +39,13 @@ const ModuleCard = ({
 		// eslint-disable-next-line camelcase
 		required_actions,
 	} = neveDash.modules[slug];
-	const { upgradeLinks } = neveDash;
+	const { upgradeLinks, autoModuleConfigAvailable } = neveDash;
+
+	const disabledReason = autoModuleConfigAvailable
+		? __(
+				'Cannot be updated due to automatic module configurations mode is enabled.'
```

Comment:
I would suggest rephrasing this to "Cannot be changed due to automatic module configurations mode being enabled."

Reply:
Thank you for your help!